tests/generator/not_impl_modules_footer.txt

```diff
@@ -2,39 +2,27 @@
 
 def dir_of_mod_or_default(module, default=None):
     try:
-        mod=set(dir(__import__(module)))
+        mod = set(dir(__import__(module)))
         return mod
-    except:
+    except (ImportError, AttributeError):
         return default
 
-rustpymods = list(
-    map(
-        lambda mod: mod[0],
-        filter(
-            lambda mod: (mod[1] == "" or mod[1] == ".py") and "LICENSE" not in mod[0],
-            map(os.path.splitext, os.listdir(libdir)),
-        ),
-    )
-)
-rustpymods += list(sys.builtin_module_names)
-
-
-not_imported=set('not imported')
-rustpymods = dict(map(
-    lambda mod: (
-        mod,
-        dir_of_mod_or_default(mod, not_imported)
-        if mod not in ("this", "antigravity") else None
-    ),
-    rustpymods
-))
+mod_names = [
+    name.decode() for name, ext in map(os.path.splitext, os.listdir(libdir))
```

Comment:
The bugfix is `name.decode()`. `os.listdir` returns list of bytes here but the code assumes it was str. Calling `decode()` will prevent future breaking by similar reason